src/lib/views/studies/StudyUsageTooltip.svelte

```diff
@@ -0,0 +1,104 @@
+<script>
+  let clazz;
+  export { clazz as class };
```

Comment:
Really liking this and will use it from now on.

Reply:
Yeah its neat!